README.md

```diff
@@ -1,2 +1,17 @@
 # hw01
 Repository for first CFSS homework assignment
+
+#About Me
+Hi! My name is **Di Zhou**. I'm a MAPSS student concentrating in sociology. My research interests are political socialization and social change. 
+
+I have three cats at home! They are:
+* Mountain
+* Momo
+* Coco
+
+![a picture of coco](coco-insta.png)
```

Comment:
Thank you for your review Nora! Btw my instagram name is "dizhou". Let's get connected 😆